versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md

```diff
@@ -199,6 +199,31 @@ For running integration test:
 SEQ_ACCOUNT=myAccount SEQ_USER=myUser SEQ_PW=myPassword SEQ_ROLE=myRole SEQ_DB=myDatabaseName SEQ_SCHEMA=mySchema SEQ_WH=myWareHouse npm run test-integration-snowflake
 ```
 
+### Oracle Database
+
+The underlying connector library used by Sequelize for Oracle is the [node-oracledb](https://www.npmjs.com/package/oracledb) package.  
+See [Releases](/releases#oracle-support-table) to see which versions of Oracle Database & node-oracledb are supported.
+
+node-oracledb needs IC[Oracle Instant Client](https://www.oracle.com/database/technologies/instant-client/downloads.html) to work. You can use node-oracledb [quick start](https://oracle.github.io/node-oracledb/INSTALL.html#quickstart) link for installations.
```

Comment:
Don't use "IC" - just say Oracle Instant Client.  

In the 2nd sentence add "the" before "node-oracledb"

Reply:
incorporated the changes.